src/library.js

```diff
@@ -3194,7 +3194,7 @@ LibraryManager.library = {
   $setWasmTableEntry__deps: ['$wasmTableMirror'],
   $setWasmTableEntry: function(idx, func) {
     wasmTable.set(idx, func);
-    wasmTableMirror[idx] = func;
+    wasmTableMirror[idx] = wasmTable.get(idx);
```

Comment:
It might be good to add a comment here, as it is not obvious when reading the code (and I worry a future refactor might "optimize" this).